### ISSUE TITLE
After Build, App has a Unreachable Express Server (http://localhost:3000/) 

### ISSUE THREAD

**Issue by JojoMig** [chris-schmitz/electron-express-example#1]
Hi there, after running the Build Command, and using the app exe under "electron-express-win32-x64" , I'm getting the "This site can’t be reached". Cant figure out why the connection is being refused as I'm using an updated dev ENV. 

The package was done with  electron v1.7.6.
> Packaging app for platform win32 x64 using electron v1.7.6

PS: If Serving In "run dev" works fine.

**Comment by chris-schmitz** [comment]
Hey, sorry I missed this message. I see you closed the issue, did you figure it out?

**Comment by JojoMig** [comment]
> 
> 
> Hey, sorry I missed this message. I see you closed the issue, did you figure it out?

Hi @chris-schmitz, sorry for the lack of response at the closing time, guess I was shamed :P the issue was due to the Comodo Suite I use. It was for some weird reason blocking access to the :3000 port when running the ".exe", never quite figured out why that occurs only for the ".exe" and not on the dev environment, my work build is quite confusing to debug this issue, so after adding the rule to the Comodo for both the ".exe" and the "URL", I managed to run the Express Server. At the time I thought it was messing with the child process.. Your code was very useful though. TY
Cheers

**Comment by JojoMig** [comment]
> 
> Hey, sorry I missed this message. I see you closed the issue, did you figure it out?

Hi @chris-schmitz, It's ok man, sorry for the lack of response at the closing time, guess I was ashamed :P the issue was due to the Comodo Firewall Suite I use. It was for some weird reason blocking access to the :3000 port when running the ".exe", never quite figured out why that occurs only for the ".exe" and not on the dev environment, my work build is quite confusing to debug this issue, so after adding the rule to the Comodo for both the ".exe" and the "URL", I managed to run the Express Server. At the time I thought it was messing with the child process.. Your code was very useful to better understand the Child-Process though. TY
Cheers
